app/src/main/java/com/jerboa/ui/components/home/BottomNavActivity.kt

```diff
@@ -308,6 +311,7 @@ fun BottomNavActivity(
                             siteViewModel = siteViewModel,
                             drawerState = drawerState,
                             blurNSFW = appSettings.blurNSFW.toEnum(),
+                            padding = padding,
```

Comment:
Bit annoying, at some point I'll see if I can lift this all out of this nested navhost and just have a single one